sdk/python/kfp/components/structures.py

```diff
@@ -74,60 +77,126 @@ class OutputSpec(base_model.BaseModel):
     description: Optional[str] = None
 
 
-class InputValuePlaceholder(base_model.BaseModel):
+class PlaceholderSerializationMixin(abc.ABC):
```

Comment:
For my education, does this need to be abstract class? Don't see we use anything that needs to be abstract?

Reply:
Good catch -- this is not doing anything. Removed!